doc/setup.md

```diff
@@ -49,14 +49,23 @@ If you are using Windows Subsystem for Linux (WSL) be sure you clone the reposit
 
 These instructions assume that you have [Homebrew](http://brew.sh/) installed.
 
-1. Install dependencies
+1. Install project dependencies
 
-	```
-	brew install supervisor nginx postgresql node \
-	     llvm libomp gsl rust
-	nvm install node
-	```
-	- If you want to use [brotli compression](https://en.wikipedia.org/wiki/Brotli) with NGINX (better compression rates for the frontend), you can install NGINX with the `ngx_brotli` module with this command: `brew tap denji/nginx && brew install nginx-full --with-brotli`. _If you already had NGINX installed, you may need to uninstall it first with `brew unlink nginx`._ Otherwise, you can install NGINX normally with `brew install nginx`.
+Using Homebrew, install core dependencies:
+```
+brew install supervisor nginx postgresql node llvm libomp gsl rust
+```
+Then, install Node.js with NVM:
+```
+nvm install node
+```
+Finally, install these compression libraries. These are needed in order to install the Python dependency `tables` later on:
+```
+brew install hdf5 c-blosc lzo bzip2
+```
+After installing each package, Homebrew will print out the installation paths. You should add these paths to your `.zshrc` file to ensure SkyPortal can locate these libraries. Instructions for this can be found in the [Configuring Shell Environment for Development](#configure-shell-mac) section below.
+
+If you want to use [brotli compression](https://en.wikipedia.org/wiki/Brotli) with NGINX (better compression rates for the frontend), you can install NGINX with the `ngx_brotli` module with this command: `brew tap denji/nginx && brew install nginx-full --with-brotli`. _If you already had NGINX installed, you may need to uninstall it first with `brew unlink nginx`._ Otherwise, you can install NGINX normally with `brew install nginx`.
```

Comment:
Can you just move that back up to right after where nginx is installed? Makes more sense to have that here than there